src/datastore/postgres-store.ts

```diff
@@ -4345,6 +4345,114 @@ export class PgDataStore
     });
   }
 
+  async getInformationTxsWithStxTransfers({
+    stxAddress,
+    tx_id,
+  }: {
+    stxAddress: string;
+    tx_id: string;
+  }): Promise<DbTxWithStxTransfers> {
+    return this.query(async client => {
+      const queryParams: (string | Buffer)[] = [stxAddress, hexToBuffer(tx_id)];
+      const resultQuery = await client.query<
+        TxQueryResult & {
+          count: number;
+          event_index?: number;
+          event_type?: number;
+          event_amount?: string;
+          event_sender?: string;
+          event_recipient?: string;
+        }
+      >(
+        `
+      SELECT 
```

Comment:
Various new microblock related properties need to be included in this query, see https://github.com/blockstack/stacks-blockchain-api/blob/ab45a151f4136a1e8907befc1d6047035f6fd6f8/src/datastore/postgres-store.ts#L4380-L4420

Reply:
Changes have been made.
I have added microblock_canonical = true into the query, where required.